manager/status_handler.py

```diff
@@ -58,12 +58,12 @@ def handle_patch(cls, **kwargs):
             if rows_modified == 0:
                 # sysid/cve/acct combination does not exist
                 return cls.format_exception('inventory_id/cve must exist and inventory_id must be visible to user', 404)
-        except (IntegrityError, ValueError) as integ_error:
+        except (IntegrityError, psycopg2IntegrityError) as value_error:
```

Comment:
what is this change doing?

Reply:
There are 3 commits in the PR and this change is in a commit labeled "fixed PatchStatus handler due to peewee update".  I recall seeing a previous PR that included a psycopg2.IntegrityError as 'psycopg2IntegrityError' and added it to an except clause along with the regular IntegrityError.  I don't know what changed in peewee to require that, but this commit looks to be fixing an unintended switch of the ValueError except clause and the clause meant to handle both IntegrityError's.  @Josca will have to confirm this.

Reply:
This is fix of `test_bad_status_id`. For previous used version of peewee (3.9.3), exception `psycopg2IntegrityError` was thrown, with message:

`insert or update on table "system_vulnerabilities" violates foreign key constraint "status_id"
DETAIL:  Key (status_id)=(19) is not present in table "status".`

For the last version of peewee (3.10.0) exception with the exactly same message is thrown but of different type (peewee.IntegrityError) so I had to modify exceptions catching this way.

Btw, I think that cathing db integrity errors in the same section is more logical.

Reply:
Also maybe we should change line 66:
~~~python
except ValueError as value_error
~~~
to
~~~python
except Exception as ex
~~~
.. to ensure db rollback for each possible error.

Reply:
ok, thank you for explanation

Reply:
yw

Reply:
about changing to `except Exception as ex`, I'm ok with that, let's create separate PR.